demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -59,13 +59,20 @@ const ContentScopeProvider: React.FC<Pick<ContentScopeProviderProps, "children">
     const allowedSiteConfigs = Object.fromEntries(
         Object.entries(sitesConfig.configs).filter(([siteKey, siteConfig]) => allowedUserDomains.includes(siteKey)),
     );
-    const values: ContentScopeValues<ContentScope> = {
-        domain: Object.keys(allowedSiteConfigs).map((key) => ({ value: key })),
-        language: [
-            { label: "English", value: "en" },
-            { label: "German", value: "de" },
-        ],
-    };
+    const values: Array<ContentScopeInterface> = [
```

Comment:
Using this type isn't correct. `ContentScopeInterface` denotes a concrete content scope, e.g., `{ "domain": "main", language: "en" }`. Please create a separate type for the values.